### DISCUSSION TITLE
How reliable to use Single Node Multiple Drivers mode?

### DISCUSSION THREAD

**Discussion by link89** [minio/minio#18261] [Q&A] [answered]
I am investigating using minio for research data management.  Currently we plan to use a single tower server with a 120G SSD for operating system, and 12T HDD x 2 for minio, and basically we have 2 options:

1. Use 12T HDD x 2 to build a single node multiple drivers minio instance.
2. Use 12T HDD x 1 to build a signle node single driver instance, and the other disk as a backup storage that backup data in minio every day.

We don't need to ensure the minio that has the ability of 7 x 24 online, downtime is acceptable, but data loss is not, we need to ensure we can reovery data when any of the driver is broken. 

What would happen to a minio single node multiple drivers instance when the operating system SSD is broken? Will data get loss in such case? If not, how can I recover from such failure?

**Comment by klauspost** [comment, marked as answer]
> We don't need to ensure the minio that has the ability of 7 x 24 online, downtime is acceptable, but data loss is not, we need to ensure we can reovery data when any of the driver is broken.

For this case, it will be fine. You will with default settings be able to lose 4 drives and still have read access to the data and write once at least one drive is added back.

> What would happen to a minio single node multiple drivers instance when the operating system SSD is broken? Will data get loss in such case? If not, how can I recover from such failure?

Everything is stored on data drives, so once you have restored the system and reinstalled minio with similar config, it will continue to operate.
